views/core/campaigninsightsdailyagg_rfn.view.lkml

```diff
@@ -104,16 +85,60 @@ view: +campaigninsightsdailyagg {
     description: "The number of likes of your Facebook Page, attributed to your ads."
     sql: ${page_likes} ;;
   }
-  measure: average_of_cpe{
-    type: average
+  measure: sum_of_video_p95_watched_actions_video_views {
+    type: sum
+    value_format_name: "positive_m_or_k"
+    description: "The number of times your video was played at 95% of its length, including plays that skipped to this point."
+    sql: ${video_p95_watched_actions_video_views} ;;
+  }
+  measure: sum_of_link_clicks {
+    type: sum
+    value_format_name: "positive_m_or_k"
+    description: "The number of clicks on links within the ad that led to advertiser-specified destinations, on or off Meta technologies."
+    sql: ${link_clicks} ;;
+  }
+
+  #A measures with calculations
+  measure: cpm{
+    type: number
+    value_format_name: usd
+    description: "The average cost for 1,000 impressions."
+    sql: SAFE_DIVIDE(${sum_of_total_spend}, ${sum_of_total_impressions} / 1000) ;;
+  }
+  measure: engagement_rate{
+    type: number
+    value_format:"0.00\%"
+    description: "The total engagement divided by total reach."
+    sql: SAFE_DIVIDE( ${sum_of_post_engagement}, ${sum_of_total_reach}) * 100 ;;
+  }
+  measure: link_ctr{
+    type: number
+    value_format:"0.00\%"
+    description: "The number of clicks that your ad receives divided by the number of times your ad is shown."
+    sql: SAFE_DIVIDE( ${sum_of_link_clicks}, ${sum_of_total_impressions}) * 100 ;;
+  }
+  measure: cpe{
+    type: number
     value_format:"0.00\%"
     description: "Compares the cost to the total amount of engagements on a Facebook post that include: likes and reactions. comments. shares."
-    sql: ${cpe} ;;
+    sql: SAFE_DIVIDE( ${sum_of_total_spend}, ${sum_of_post_engagement}) ;;
   }
-  measure: average_of_cpcv{
-    type: average
+  measure: cpcv{
+    type: number
     value_format_name: usd
-    description: "Pay for a video ad once the user watches a video in its entirety"
-    sql: ${cpcv} ;;
+    description: "Pay for a video ad once the user watches a video in its entirety."
+    sql: SAFE_DIVIDE( ${sum_of_total_spend}, ${sum_of_video_p95_watched_actions_video_views}) ;;
+  }
+  measure: frequency {
+    type: number
+    value_format_name: "positive_m_or_k"
+    description: "The average number of times each person saw your ad. This metric is estimated."
+    sql: SAFE_DIVIDE( ${sum_of_total_impressions}, ${sum_of_total_reach}) ;;
+  }
+  measure: vtr{
+    type: number
+    value_format:"0.00\%"
```

Comment:
@KateMordasova Could you try using `value_format_name: percent_1` this should let you skip the * 100

Reply:
@pthoutam-googler Yes, sure, I will fix it.